app/models/registration.rb

```diff
@@ -293,6 +293,7 @@ def registration_history
 
   def to_v2_json(admin: false, history: false, pii: false)
     base_json = {
+      user: user,
```

Comment:
Seems like you're sneaking in a feature addition?

Reply:
Ah, now I see why you're doing this. It saves you one `getUserInfo` call in the frontend.

Reply:
Although I do want to point out that serializing a user is quite "expensive", I think we include stuff like team membership by default (for legacy reasons). This will definitely slow down the loading even more.

Reply:
true, that's why we removed a bunch of stuff in the users info route, so I will make sure it also includes only the data we needed